mmcv/runner/hooks/profiler.py

```diff
@@ -131,6 +132,15 @@ def _log_handler(prof):
                     raise ImportError('please run "pip install '
                                       'torch-tb-profiler" to install '
                                       'torch_tb_profiler')
+                if 'dir_name' not in trace_cfg:
+                    trace_cfg['dir_name'] = osp.join(runner.work_dir,
+                                                     'tf_tracing_logs')
```

Comment:
Hi @GPhilo, the `tf` in `tf_tracing_logs` make no sense. We should use a better name e.g. `tb_traceing_logs`. What do you think?

Reply:
Hey! I agree that probabily `tb` would be more accurate, but this was the suggested name from @zhouzaida and at the moment the [TensorboardLogger also uses te same naming scheme](https://github.com/open-mmlab/mmcv/blob/b58c84c6f0e156ed88646d0679b15e693283cf59/mmcv/runner/hooks/logger/tensorboard.py#L55), which is why `tf_` was suggested in the first place.

Reply:
Cool, that is ok with me!  ^_^